### DISCUSSION TITLE
Is a bookmarklet to Shlink possible?

### DISCUSSION THREAD

**Discussion by socal0001** [shlinkio/discussions#135] [Help wanted] [answered]
### Shlink version

3.7.0

### PHP version

8.2

### How do you serve Shlink

Openswoole Docker image

### Database engine

PostgreSQL

### Database version

15.5-1

### Summary

While browsing the web on my desktop, I'd like to be able to click on a bookmark on my tooltool bar that will create a Shlink.  Would a bookmarklet work: https://en.wikipedia.org/wiki/Bookmarklet ? I don't have the skills/knowledge to know or to do it myself.

Ideally I would have two bookmarklets, one would create a link with a random slug.  The other would open shlink-web-client with the link pasted in to allow me to custome the slug, title, etc.

Your thoughts, please.

**Comment by acelaya** [comment, marked as answer]
I think it should be possible, using the API.

Check [the spec](https://api-spec.shlink.io/) and the [documentation](https://shlink.io/documentation/api-docs/)

**Reply by socal0001** [reply]
Thanks, not sure I have the skills for that.  I found this webpage that goes over how.

If someone does this, I recommend it be added to the install/documments/features as it would be a handy way to speed up workflow.  

https://www.freecodecamp.org/news/what-are-bookmarklets/
